### DISCUSSION TITLE
Labels from Labeler not getting shown in App - Potential delay on Label Propagation Question

### DISCUSSION THREAD

**Discussion by MrMysterius** [bluesky-social/atproto#3242] [General Discussion]
> *Not sure on the topic to choose, so I just took general one...*

Hi I have made a labeler that flags stuff based on keywords there are a lot of labels and a lot of keyword. The problem I am facing now and I don't know if I am doing something wrong or it is some gimmicky bsky backend quirk. But the labeler itself labels posts instantly basically after they have been posted, and I can manually hit the labeler route for a post and get the actual labels. Now the problem is it is not showing in the App for people that have subscribed to the labeler, some posts for testing had the labels showing up after hours... but if that is the default it is pretty much useless for me and the ones subscribing...

Tried to find stuff on it, but I have not found anything that would tell me if I'm doing something wrong or bsky is doing something unexpected...

Also the rate at which these labels are applied on my labeler is pretty high like 5-20 labels per second... and if anyone wondering what for it is for general kinks, since some people might want the granularity that the labeling system brings while also blocking stuff they don't linke.

**Comment by MrMysterius** [comment]
Found out there's a rate limit on labeler ingestion for the AppView

**Reply by aarondodd** [reply]
Could you elaborate? I'm experiencing the same thing but can't find a good doc on what the limit is.

Thanks

**Reply by MrMysterius** [reply]
I think the limit daily was 50k labels per labeler. There are second and hourly/minutely limits too but nott so sure in them anymore (got this info from a discord)
